ddsx11/tests/latency/sender/main.cpp

```diff
@@ -8,7 +8,14 @@
  * @copyright Copyright (c) Remedy IT Expertise BV
  */
 
+#if DDSX11_NDDS==1
+#include "dds/ndds_typedefs.h"
+#else
+#include "dds/opendds_typedefs.h"
+#endif
 #include "latency_data_dds_typesupport.h"
+#include "latency_data_ddsSupport.h"
+#include "latency_data_dds_traits.h"
```

Comment:
`latency_data_ddsSupport.h` is a RTI specific file so shouldn't this be together with line 12?

Reply:
Yes, just had the compile error with OpenDDS. Updated the code.